jsapp/js/components/formViewTabs.es6

```diff
@@ -64,6 +64,7 @@ class FormViewTabs extends Reflux.Component {
 
   triggerRefresh(evt) {
     if ($(evt.target).hasClass('active')) {
+      // ROUTES.FORM_RESET
```

Comment:
I added this comment for searchability, since removing this section was missed [here](https://github.com/kobotoolbox/kpi/pull/4505#discussion_r1238709429)

Reply:
Ok. I guess this is why django has that reverse function.